app/src/androidTest/java/ch/sdp/vibester/activity/ScoreBoardActivityTest.kt

```diff
@@ -22,12 +23,22 @@ class ScoreBoardActivityTest {
     val activityRule = ActivityScenarioRule(ScoreBoardActivity::class.java)
 
     @Test
-    fun recycleViewToViewTest() {
+    fun genreShouldDisappearAfterSelected() {
+        onView(withId(R.id.genrePerScoreboard)).check(matches(isDisplayed()))
+        onView(withId(R.id.rockButton)).perform(click())
+        onView(withId(R.id.genrePerScoreboard)).check(matches(not(isDisplayed())))
+    }
+
+    @Test
+    fun rockBtnShouldSetUpRecycleVie() {
```

Comment:
```suggestion
    fun rockBtnShouldSetUpRecycleView() {
```

Reply:
Does this typo still exist? I remember fixing it in commit 5b5424048dd2b1cea7e59e02053d5b718bcffc13